### PR TITLE
rebuild videos page

### PR DESCRIPTION
**Pull request by Dzmitry92**
What I've done:
 * rebuild videos page
 * added header
 * added paginator

### CODE REVIEW

File: packages/web-app/web-app-front/src/index.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { createRoot } from 'react-dom/client';
-import App from './app/app';
+import App from './app/App';
```
Comment (SerafimovichEugene) [inline packages/web-app/web-app-front/src/index.tsx:3]: import App from './app/app'.

Comment (SerafimovichEugene) [reply packages/web-app/web-app-front/src/index.tsx:3]: It works on mac and win but it crashes on linux.

File: packages/web-app/web-app-front/src/components/GlobalHeader/styles.css [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.header-container {
+  margin-bottom: 20px;
+  padding: 0 !important;
```
Comment (SerafimovichEugene) [inline packages/web-app/web-app-front/src/components/GlobalHeader/styles.css:3]: Please don't use `!important`;

File: packages/web-app/web-app-front/src/components/Spinner/index.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+export { Spinner } from './Spinner';
```
Comment (SerafimovichEugene) [inline packages/web-app/web-app-front/src/components/Spinner/index.ts:1]: I don't see why we should reexport files from index.ts. Please delete these re-exporting index files.

File: packages/web-app/web-app-front/src/data/api/index.ts [hunk 4]
```diff
@@ -0,0 +1 @@
+export { useVideos } from './api';
```
Comment (SerafimovichEugene) [inline packages/web-app/web-app-front/src/data/api/index.ts:1]: same

File: packages/web-app/web-app-front/src/pages/VideosPage/VideoCard/VideoCard.tsx [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+type VideoCardType = {
+  link: string;
+  name: string
+}
+
+export const VideoCard = (props: VideoCardType) => {
+  const { link, name } = props;
+  return (
+    <Link to={link} className="card">
+      <img src="https://via.placeholder.com/140x100" className="card-img-top" alt="img" />
```
Comment (SerafimovichEugene) [inline packages/web-app/web-app-front/src/pages/VideosPage/VideoCard/VideoCard.tsx:13]: hard src link to placeholder is not accepted. Please add any stub image locally.

File: packages/web-app/web-app-front/src/pages/VideosPage/styles.css [hunk 6]
```diff
@@ -0,0 +1,30 @@
+.grid-container-video-cards {
+  margin-bottom: 20px;
+  width: 100%;
+  display: grid;
```
Comment (SerafimovichEugene) [inline packages/web-app/web-app-front/src/pages/VideosPage/styles.css:4]: `display: grid;` should come the first.

File: packages/web-app/web-app-front/src/pages/VideosPage/styles.css [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+  grid-template-columns: repeat(auto-fill, minmax(260px, 1fr));
+  gap: 16px;
+}
+
+.container-main {
+  height: calc(100vh - 80px);
+  display: flex;
```
Comment (SerafimovichEugene) [inline packages/web-app/web-app-front/src/pages/VideosPage/styles.css:11]: same

**Review by SerafimovichEugene — CHANGES_REQUESTED** [review]

**Comment by SerafimovichEugene** [comment]
Paginator is not responsive

File: packages/web-app/web-app-front/src/index.tsx (hunk 1 above)
Comment (Dzmitry92) [reply packages/web-app/web-app-front/src/index.tsx:3]: fixed

**Review by SerafimovichEugene — APPROVED** [review]

File: packages/web-app/web-app-front/src/pages/VideosPage/VideosPage.tsx [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+import { VideoCard } from "./VideoCard/VideoCard";
+import { FileEntity } from "../../domain/index";
+import { Spinner } from "../../components/Spinner/Spinner";
+import { useVideos } from "../../data/api/api";
+import { Paginator } from "../../components/Paginator/Paginator";
+import { DEFAULT_PAGE, DEFAULT_LIMIT } from "../../constants";
+import "./styles.css";
+
+const VideosPage = () => {
+  const [page, setPage] = useState(DEFAULT_PAGE);
```
Comment (SerafimovichEugene) [inline packages/web-app/web-app-front/src/pages/VideosPage/VideosPage.tsx:12]: hello

**Pull request merged**